### PR TITLE
Updates callsites for get_prs_list

### PR DESCRIPTION
**Pull request by JavonDavis**
Updates callsites for get_prs_list to use new implementation

To test: 

1. Run `bundle exec fastlane get_pullrequests_list milestone:<milstone>` to generate a PRs list file and validate the contents
2. Run `bundle exec fastlane code_freeze` and validate the contents of the prs list file generated

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -374,7 +377,7 @@ platform :android do
   ########################################################################
   desc "Get a list of pull request from `start_tag` to the current state"
   lane :get_pullrequests_list do | options |
-    get_prs_list(repository:GHHELPER_REPO, start_tag:"#{options[:start_tag]}", report_path:"#{File.expand_path('~')}/wcandroid_prs_list.txt")
+    get_prs_list(repository:GHHELPER_REPO, milestone:"#{options[:milestone]}", report_path:"#{File.expand_path('~')}/wcandroid_prs_list.txt")
   end
```
Comment (mokagio) [inline fastlane/Fastfile:381]: I was going to suggest to use this lane in the `code_freeze` to DRY the code a bit, but I'm not sure what a good way to provide the file name could be 🤔 

If the only parameter that we'd be sparing is `repository` then maybe there's not much value in  having more abstraction in this lane...

**Review by mokagio — APPROVED** [review]
I tested using [this commit](https://github.com/woocommerce/woocommerce-android/commit/44bd628dc40a011df7639abb663269f00c81f644) and by commenting out the Git calls in `release-toolkit` that would have have pushed a release branch from the `code_freeze` lane.

I run the lanes as suggested and verified the result against the milestone view here in GitHub.

This is out of scope, but I noticed the list generated by `get_prs_list` included open PRs as well. I'm not sure whether that's appropriate, though, at least in the context of the code freeze.

**Comment by JavonDavis** [comment]
> I'm not sure whether that's appropriate, though, at least in the context of the code freeze.

Yep it's intended!

File: fastlane/Fastfile (hunk 1 above)
Comment (JavonDavis) [reply fastlane/Fastfile:381]: > If the only parameter that we'd be sparing is repository then maybe there's not much value in having more abstraction in this lane...
Yeah I think we can go ahead and work with it as is for this case

**Pull request merged**